linkage/pvs_like_case_study/pvs_like_case_study_sample_data.ipynb

```diff
@@ -15,26 +15,161 @@
    "metadata": {},
```

Comment:
<p>cell above this one is not appearing in reviewnb, but if it was, I would query whether you meant to have term_frequency_adjustments=True.   I thought this was a splink invention that was not present in SASMatch.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Good point, I've removed this in the next version for realism.